CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# [0.7.0](https://github.com/mattrglobal/node-bbs-signatures/compare/0.6.0...0.7.0) (2020-05-04)
+
```

Comment:
Thanks are you able to amend the changelog with a comment here saying Support for Node 11 and 12 added

Reply:
fixed, but wait until CI completes to merge because I forced pushed the changes on with a rebase so I don't know how the the tagging will react.